factory/grpcServerFactory.go

```diff
@@ -0,0 +1,33 @@
+package factory
+
+import (
+	"fmt"
+
+	"github.com/multiversx/mx-chain-ws-connector-firehose-go/config"
+	"github.com/multiversx/mx-chain-ws-connector-firehose-go/process"
+	"github.com/multiversx/mx-chain-ws-connector-firehose-go/server"
+)
+
+// CreateGRPCServer will create the gRPC server along with the required handlers.
+func CreateGRPCServer(
```

Comment:
this is not used anymore.. is it still needed? if so, we should return a disabled component if it is not enabled, instead of nil